src/website/app/demos/Button/examples/index.js

```diff
@@ -15,36 +15,26 @@
  */
 
 /* @flow */
-import button from './button';
+import regular from './regular';
 import circular from './circular';
-import fullWidth from './fullWidth';
-import danger from './danger';
-import disabled from './disabled';
 import icons from './icons';
 import iconOnly from './iconOnly';
 import minimal from './minimal';
 import primary from './primary';
 import rtl from './rtl';
 import sizes from './sizes';
 import states from './states';
-import success from './success';
 import truncation from './truncation';
-import warning from './warning';
 
 export default [
-  button,
+  states,
```

Comment:
Why is `states` at the top?  The intent was to originally hide this from production and only keep it for dev/test.  If it is at the bottom, then when the page is loaded during development, it gives a close resemblance to that of what the user will see.